python/lsst/meas/astrom/ref_match.py

```diff
@@ -27,6 +27,7 @@
 import lsst.pex.config as pexConfig
 import lsst.pipe.base as pipeBase
 from .matchOptimisticB import MatchOptimisticBTask
+from .matchPessimisticB import MatchPessimisticBTask
```

Comment:
Right, this was left over from when I was testing and before I discovered the retarget command in config files. I have removed the MatchPessimisticBTask import.